content/doc/book/installing/_kubernetes.adoc

```diff
@@ -0,0 +1,524 @@
+
+. Now you can install Jenkins by running the `helm install` command and passing it the
+  following arguments:
+
+  * The name of the release `jenkins`
+  * The -f flag with the YAML file with overrides `jenkins-values.yaml`
+  * The name of the chart `jenkinsci/jenkins`
+  * The `--namespace` flag with the name of your namespace `jenkins`
+  +
+  [source,bash]
+  ----
+  helm install jenkins -f jenkins-values.yaml jenkinsci/jenkins --namespace jenkins
+  ----
+
+  This outputs something similar to the following:
+  +
+  [source,bash]
+  ----
+  NAME: jenkins
+  LAST DEPLOYED: Wed Sep 16 11:13:10 2020
+  NAMESPACE: jenkins
+  STATUS: deployed
+  REVISION: 1
+  NOTES:
+  1. Get your 'admin' user password by running:
+  printf $(kubectl get secret --namespace jenkins jenkins -o jsonpath="{.data.jenkins-admin-password}" | base64 --decode);echo
+  2. Get the Jenkins URL to visit by running these commands in the same shell:
+  export NODE_PORT=$(kubectl get --namespace jenkins -o jsonpath="{.spec.ports[0].nodePort}" services jenkins)
+  export NODE_IP=$(kubectl get nodes --namespace jenkins -o jsonpath="{.items[0].status.addresses[0].address}")
+  echo http://$NODE_IP:$NODE_PORT/login
+  3. Login with the password from step 1 and the username: admin
+  4. Use Jenkins Configuration as Code by specifying configScripts in your values.yaml file, see documentation: http:///configuration-as-code and examples:  https://github.com/jenkinsci/configuration-as-code-plugin/tree/master/demos
+  +
+  For more information on running Jenkins on Kubernetes, visit:
+  https://cloud.google.com/solutions/jenkins-on-container-engine
+  For more information about Jenkins Configuration as Code, visit:
+  https://jenkins.io/projects/jcasc/
+  ----
+
+. Depending on your environment, it can take a bit of time for Jenkins to start up. Enter the
+  following command to inspect the status of your Pod:
++
+[source,bash]
+----
+kubectl get pods --namespace=jenkins
+----
+
+  Once Jenkins is installed, the status should be  set to Running as in the following output:
++
+[source,bash]
+----
+❯ kubectl get pods --namespace=jenkins
+NAME                       READY   STATUS    RESTARTS   AGE
+jenkins-645fbf58d6-6xfvj   1/1     Running   0          2m
+----
+
+. To access your Jenkins server, you must retrieve the password. You can retrieve your password
+  using either of the two options below.
++
+*Option 1*
++
+Run the following command:
++
+[source,bash]
+----
+printf $(kubectl get secret --namespace jenkins jenkins -o jsonpath="{.data.jenkins-admin-password}" | base64 --decode);echo
+----
++
+The output should look like this:
++
+[source,bash]
+----
+Um1kJLOWQY
+----
++
+[NOTE]
+====
+👆🏻Note that your password will be different.
+====
++
+*Option 2*
++
+Run the following command:
++
+[source,bash]
+----
+kubectl get secret --namespace jenkins jenkins -o jsonpath="{.data.jenkins-admin-password}"
+----
++
+The output should be a base64 encoded string like this:
```

Comment:
I just feel like this is something that should be strongly highlighted because it's easy to just think that it's a simple copy-paste like above.
```suggestion
The output should be a **base64 encoded string** like this:
```